binr/rax2/rax2.c

```diff
@@ -228,8 +229,60 @@ static int rax(char *str, int len, int last) {
 		return true;
 	}
 	if (flags & (1 << 2)) { // -S
-		for (i = 0; i < len; i++) {
-			printf ("%02x", (ut8) str[i]);
+		for (int i = 0; i < len; i++) {
+			if (str[i] == '\\') {
+         
+				++i;
+				if (str[i] == 'n') printf("%02x", 10);
```

Comment:
use a switch table and refactor this into a separate function that returns the dec value and just use a single printf statement